src/mixins/crownConfig.js

```diff
@@ -225,16 +225,17 @@ export default {
     updateCrownPosition() {
       const buttonLength = 25;
       const buttonMargin = 10;
+      const { sx, sy } = this.paper.scale();
       const { x, y, width, height } = this.shape.findView(this.paper).getBBox();
       const { tx, ty } = this.paper.translate();
       const crownHeight = (buttonLength * this.buttons.length) + (buttonMargin * (this.buttons.length - 1));
-      const centerY = 0 - (crownHeight / 2) + (height / 2);
+      const centerY = sy / (0 - ((crownHeight / 2) + (height / 2)));
```

Comment:
`centerY` is no longer the "centre" position anymore since you're aligning it to the top; this can simply be `0`. And if that's the case, a few lines can be removed and the whole thing can be:

```js
updateCrownPosition() {
  const buttonLength = 25;
  const buttonMargin = 10;
  const { sx, sy } = this.paper.scale();
  const { x, y, width } = this.shape.findView(this.paper).getBBox();
  const { tx, ty } = this.paper.translate();

  this.buttons.forEach((button, index) => {
    const yOffset = ((buttonLength + buttonMargin) * index) * sy;

    button.resize(buttonLength, buttonLength);
    button.position((x + width + buttonMargin - tx) / sx, (y + yOffset - ty) / sy);
  });
},
```